sonic-net/sonic-buildimage
# Fix Belgite platform issues

#### Why I did it
Harden the Belgite platform and fix reported issues

#### How I did it
Fixed the issues in PSU, FAN, Temperature, LED and Port modules

#### How to verify it
Verified by loading the image in Belgite platform and confirm that click platform command works
Verified that pmon and its tasks up and running

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

As part of platform hardening this commit fixes several platform issues in various components like PSU, FAN, Temperature, LED.

#### A picture of a cute animal (not mandatory but encouraged)

![image](https://user-images.githubusercontent.com/29372802/212863854-38c3643c-5521-4cf5-9fa8-11d81a424999.png)
